node/directives/checkAdminAccess.ts

```diff
@@ -16,68 +16,116 @@ export class CheckAdminAccess extends SchemaDirectiveVisitor {
       info: any
     ) => {
       const {
-        vtex: { adminUserAuthToken, logger },
+        vtex: { adminUserAuthToken, storeUserAuthToken, logger },
         clients: { identity },
       } = context
 
+      // check if has admin token and if it is valid
+      const hasAdminToken = !!adminUserAuthToken
+      let hasValidAdminToken
+      // this is used to check if the token is valid by current standards
+      let hasCurrentValidAdminToken = false
```

Comment:
this `hasCurrentValidAdminToken` is the variable we use to track/apply original behavior. In the future we should stop using it and remove this variable.